ecs-cli/modules/cli/compose/entity/task/task.go

```diff
@@ -382,6 +382,14 @@ func (t *Task) buildRunTaskInput(taskDefinition string, count int, overrides map
 		}
 	}
 
+	// TODO: revert to "LATEST" when latest refers to 1.4.0
+	if ecsParams != nil && len(ecsParams.TaskDefinition.EFSVolumes) != 0 {
```

Comment:
we should only be setting the platform version when the launch type is FARGATE, otherwise EC2 tasks will fail. 
See [this PR](https://github.com/aws/amazon-ecs-cli/pull/1041) for how we implement this check and test it in the service context.